docker/usr/local/package/intel-graphics-compiler.sh

```diff
@@ -33,6 +33,26 @@ echo "Configuring using cmake file ${CMAKELISTS_FILE}"
 # Clone the dependency
 git clone https://github.com/intel/vc-intrinsics vc-intrinsics
 
+# Download the patches
+curl -L https://raw.githubusercontent.com/archlinux/svntogit-community/packages/intel-graphics-compiler/trunk/llvm12-porting.patch \
+    -o llvm12-porting.patch
+curl -L https://github.com/intel/intel-graphics-compiler/commit/c50d77f3c8a92fae6f43b47580e7d485f9ea5327.patch \
+    -o revert-stack-analysis.patch
+curl -L https://github.com/intel/intel-graphics-compiler/commit/8e1a461d3e6b85a6cf018caf6abf4a3ba9a1758d.patch \
+    -o igc-missing-limits-headers.patch
+curl -L https://github.com/intel/intel-graphics-compiler/commit/fd480731ad77213fbfcd9677d03e8243acbb306c.patch \
+    -o fix-dce.patch
+
+# Apply the patches
+patch -p1 < llvm12-porting.patch
+patch -p1 -R < revert-stack-analysis.patch
+patch -p1 < igc-missing-limits-headers.patch
+patch -p1 < fix-dce.patch
+
+# The build requires a binary in here to be run (CMCLTranslatorTool)
+# Without this directory in the path, bash can't find it
+PATH="${PATH}:$(pwd)/build/IGC/Release"
```

Comment:
I will admit this is a little hacky. [this call](https://github.com/intel/intel-graphics-compiler/blob/30cf926bc1f69761a774c2475208865099c94c4b/IGC/VectorCompiler/lib/BiF/CMakeLists.txt#L85) is the one that fails without this on the path. Welcoming better ideas here for sure.